### ISSUE TITLE
Can't set keybinding for "atom-cscope:focus-next"

### ISSUE THREAD

**Issue by davy79** [amitab/atom-cscope#3]
Hi, 

I tried setting up a keybinding for "atom-cscope:focus-next" but it seems to be ignored.
the method is available through the command pallet though.

**Comment by amitab** [comment]
To use atom-cscope:focus-next, you need to have the cscope widget open and only then you can use the keybinding to switch between the editor pane and the cscope widget. And it seems to be working for me...
Can I ask how you are trying to use it?

**Comment by davy79** [comment]
Weird, I was writing the steps to re-produce and in the process of copying the key binding I messed up my keymap.cson... (was completely blank)
I then fetched an older backup which did not have this specific keybinding, added it from scratch and now it works :smile: 

so I must have done something consistently wrong with the keybinding over the last days?

But anyway, I'm closing the ticket as it seems to work perfectly now.

Great plugin! it takes atom to another level as I was considering switching back to sublime.

There is one improvement to make though.
It would be nice to be able to generate the cscope database from within Atom.
Kinda like atom-ctags does...
